src/overview/using-the-core-library.md

```diff
@@ -32,7 +32,7 @@ The table below covers the current support for `no_std` at this moment for diffe
 [esp-wifi]: https://github.com/esp-rs/esp-wifi "Wi-Fi, BLE and ESP-NOW support"
 [esp-backtrace]: https://github.com/esp-rs/esp-backtrace "Exception and panic handlers"
 [esp-storage]: https://github.com/esp-rs/esp-storage "Embedded-storage traits to access unencrypted flash memory"
-[esp-wifi-current-support]: https://github.com/esp-rs/esp-wifi#current-support
+[esp-wifi-current-support]: https://github.com/esp-rs/esp-hal/tree/main/esp-wifi#current-support
```

Comment:
Good catch! I guess we should also update link for `esp-storage`, `esp-backtrace`, `esp-wifi`, since they now also live under `esp-hal`. Do you mind updating them? I can also update them in a separated PR.

Thanks!

Reply:
Sure. No problem.